src/test/java/de/tum/cit/aet/artemis/programming/localvcci/LocalVCSshIntegrationTest.java

```diff
@@ -170,10 +179,18 @@ private SshClient clientConnectToArtemisSshServer() throws GeneralSecurityExcept
 
         serverSessions = sshServer.getActiveSessions();
         assertThat(clientSession.isAuthenticated()).isTrue();
-        assertThat(serverSessions.size()).isEqualTo(numberOfSessions + 1);
+        assertThat(serverSessions.size()).as("There are more server sessions activated than expected.").isEqualTo(numberOfSessions + 1);
```

Comment:
This test seams to fail now

Reply:
It's still flaky, but just a _little_ bit more reliable.
I honestly don't really know how to improve this. I can only imagine that the underlying SSHD library might be opening up some additional sessions but I think it's somehow the task of the people maintaining this :)

Reply:
@BBesrour @SimonEntholzer can you please have a look?

Reply:
I will fix that in a small followup